### DISCUSSION TITLE
FNIRSI GC-01: Cannot install original firmware

### DISCUSSION THREAD

**Discussion by GeigerMuller** [Gissio/radpro#155] [Q&A] [answered]
hello
I wanted to install flood firmware on the device. But the flood firmware (version 1.6) is 62kB, but there is only 56kB of free space on the device. What can be done about it?
Now i have installed RadPro 2.0.2

![image](https://github.com/user-attachments/assets/a4de05c2-c9f5-45cf-8091-594b7d4f250b)

**Comment by Gissio** [comment, marked as answer]
Most likely you have a GC-01 device with a CH32 microprocessor. To go back to the original firmware you need to install V1.5.

**Reply by GeigerMuller** [reply]
Yes, I have it with a CH32 processor. You're right, 1.5 can fit there. Subsequently, however, 1.6 cannot be inserted because the disk space is still the same. I thought that the modified fw changes the disk layout. As far as the manufacturer is concerned, version 1.6 is available for this device.

**Reply by Gissio** [reply]
The CH32 microprocessor has 64 kB flash memory, the Geehy microprocessor, 128 kB. V1.6 only works on Geehy microprocessors.
